src/common/controllers/ads-controller.js

```diff
@@ -308,6 +317,10 @@ class AdsController extends FakeEventTarget implements IAdsController {
     }
   }
 
+  get isAdPlaying(): boolean {
```

Comment:
make it a regular function (not getter) like the other (e.g isAdBreak) 
also move it to the top where the all publics

Reply:
done